hazelcast/src/main/java/com/hazelcast/util/Clock.java

```diff
@@ -115,13 +115,7 @@ protected long currentTimeMillis() {
 
         @Override
         public String toString() {
-            final StringBuilder sb = new StringBuilder();
-            sb.append("SystemOffsetClock");
-            sb.append("{offset=").append(offset);
-            sb.append('}');
-            return sb.toString();
+            return "SystemOffsetClock" + "{offset=" + offset + '}';
```

Comment:
`SystemOffsetClock` vs. `Clock`

`"SystemOffsetClock" + "{offset="` could be merged.